src/devices/Ina226/Ina226.cs

```diff
@@ -0,0 +1,579 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Device.Model;
+using UnitsNet;
+
+namespace Iot.Device.Ina226
+{
+    /// <summary>
+    /// INA226 Bidirectional Current/Power monitor.
+    ///
+    /// The INA226 is a high-side/low-side current shunt and power monitor with an i2c compatible
+    /// interface. The device monitors both a shunt voltage drop and bus supply voltage.
+    /// Programmable calibration value, conversion times, and averaging, combined with an
+    /// internal multiplier, enable direct readouts of current in ampers an power in watts.
+    /// <see href="https://www.ti.com/lit/ds/symlink/ina226.pdf"/>
+    /// </summary>
+    [Interface("INA226 Bidirectional Current/Power monitor")]
+    public class Ina226 : IDisposable
+    {
+        private I2cDevice _i2cDevice;
+        private bool _disposeI2cDevice = false;
```

Comment:
I think we generally assume the device will always take an ownership of I2cDevice so in that regard I'd recommend to always dispose it or at least make a bool flag in the I2cDevice constructor which makes it a default and allows user to opt-out (but then, why would you ever opt-out?)